plenoptic/synthesize/mad_competition.py

```diff
@@ -365,7 +365,7 @@ def update_target(self, synthesis_target, step):
                     setattr(self, attr, torch.nn.Parameter(getattr(self, attr+'_all')[synthesis_target]))
                 else:
                     try:
-                        setattr(self, attr, getattr(self, attr+'_all')[synthesis_target].clone().to('cpu'))
+                        setattr(self, attr, getattr(self, attr+'_all')[synthesis_target].clone())
```

Comment:
Does `clone` automatically move to the cpu?

Reply:
No, it doesn't. Do you need to move to CPU here?

Reply:
Looking through the history, it looks like I added that `.to('cpu')` to get `MADCompetition` working with the plotting and animating code, so assuming those tests in `test_display.py` pass, then this is fine.  (My guess is there's some copying over to cpu that happens in the display code, which is the proper way to do it)